README.adoc

```diff
@@ -40,19 +40,8 @@ look in `..` too.
 This is a useful feature if you want to de-clutter your command line.
 
 Chronicle Core class `Jvm` loads the file `system.properties` into the System's properties.
-To ensure it is loaded early enough, add the following code to your `Main`
```

Comment:
Why did you remove this section? I think it's still relevant. And the next one too. I suggest asking @peter-lawrey